server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -227,6 +256,19 @@ public UpdateMeasureBuilder setVariations(MeasureVariations variations) {
       return this;
     }
 
+    /**
+     * Sets the variation of the updated Measure to create.
+     *
+     * @throws UnsupportedOperationException if the source measure already has a {@link MeasureVariations}
+     */
+    public UpdateMeasureBuilder setVariation(double variation) {
+      if (source.variations != null) {
+        throw new UnsupportedOperationException("Variation can not be changed if already set on source Measure");
+      }
+      this.variations = createLeakVariation(variation);
+      return this;
+    }
+
     public Measure create() {
       return new Measure(source.valueType, source.developer,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Measure"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)